### PR TITLE
Add new C# Sample app - text classification

### PR DESCRIPTION
**Pull request by sgchoi5**
- Update the address of C# API reference (/latest -> /stable)

### CODE REVIEW

**Comment by mijin85cho** [comment]
@sgchoi5 I have a question to the API URL changing.

To my understand, the "/latest" is to use the symbolic link to the latest version.
It is to reduce the manual update for the case of each API Version update,
so every document page for .NET is using "https://samsung.github.io/TizenFX/latest..."

However you updated /latest to /stable.
Are you going to use /stable for the document pages from now on?
If so, other guide pages also should update the links to API.

**Comment by sgchoi5** [comment]
WonYoung Choi recommended to use /stable/ instead of /latest/.
He said /latest/ could be broken.

**Comment by mijin85cho** [comment]
We communicated with WonYoung Choi on messenger,
to use the 'latest' as the same usage with the 'stable'.

```
[최원영 / WonYoung Choi] - 15:10 
네.. 그럼 latest -> stable -> API6 로 변경하도록 하겠습니다..
```

So, please use the 'latest' for the easy management of the API links.

**Comment by sgchoi5** [comment]
Changed /stable/ to previous /latest/

File: docs/application/dotnet/samples/machinelearning/overview.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+<!-- Tab content -->
+<div class="tabcontent" id="Mobile">
+<table>
+	<tbody>
+		<tr>
+			<td><img alt="" height="267" src="media/m62textclassification.png" width="150"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/MachineLearning/TextClassification" target="_blank"><strong>(M) TextClassification</strong></a></p>
+			<p>This sample application demonstrates how to use the nnstreamer single-shot API using <a href="https://samsung.github.io/TizenFX/stable/api/Tizen.MachineLearning.Inference.SingleShot.html" target="_blank">Tizen.MachineLearning.Inference.SingleShot</a>.</p>
+			<p>Given text are classified into predefined groups based on sentences. Pre-trained models predict whether a paragraph is positive or negative.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/machinelearning/overview.md:30]: Updated.

File: docs/application/dotnet/samples/machinelearning/overview.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+<!-- Tab content -->
+<div class="tabcontent" id="Mobile">
+<table>
+	<tbody>
+		<tr>
+			<td><img alt="" height="267" src="media/m62textclassification.png" width="150"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/MachineLearning/TextClassification" target="_blank"><strong>(M) TextClassification</strong></a></p>
+			<p>This sample application demonstrates how to use the nnstreamer single-shot API using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.MachineLearning.Inference.SingleShot.html" target="_blank">Tizen.MachineLearning.Inference.SingleShot</a>.</p>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/machinelearning/overview.md]: Updated nnstreamer to NNStreamer

**Comment by nawab-samsung** [comment]
@sgchoi5 Just an observation, the image seems distorted for me,  as I viewed the file on staging environment https://docs1.stg.tizen.org/staging/1099/application/dotnet/samples/machinelearning/overview

If the image is coming out well then kindly ignore  this comment or else I request your opinion  for the following: 
1. Is it possible to get little high resolution image, where text is legible?
I zoomed in also , but the image is not coming out well for me, kindly check & do the needful if required. 
img1

Thanks

**Comment by sgchoi5** [comment]
Changed an image with higher resolution.

File: docs/application/dotnet/samples/machinelearning/overview.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	<tbody>
+		<tr>
+			<td><img alt="" height="267" src="media/m62textclassification.png" width="150"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/MachineLearning/TextClassification" target="_blank"><strong>(M) TextClassification</strong></a></p>
+			<p>This sample application demonstrates how to use the NNStreamer single-shot API using <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.MachineLearning.Inference.SingleShot.html" target="_blank">Tizen.MachineLearning.Inference.SingleShot</a>.</p>
+			<p>Based on the given sentences, the text is classified into predefined groups. These predefined groups serve as an input to the pre-trained models to predict whether a paragraph is positive or negative.</p>
+			</td>
+		</tr>
+		</tr>
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/dotnet/samples/machinelearning/overview.md]: Removed redundant </tr>. thanks a lot!

**Comment by nawab-samsung** [comment]
@mijin85cho thanks!

**Pull request merged**